build.gradle.kts

```diff
@@ -17,7 +17,7 @@ plugins {
 
 subprojects {
     group = "io.github.lmos-ai.arc"
-    version = "0.24.0"
+    version = "0.25.0"
```

Comment:
We can use the version 0.24.0. It has not been released yet.

Reply:
done